lr-cli/cortx_setup/commands/node/prepare/config.py

```diff
@@ -103,23 +103,17 @@ def run(self, **kwargs):
                         confstore_pillar_dict[key][0],
                         value
                     )
-        # set management vip
-        # we are not updating mgmt_vip to confstore
-        # as there is no cluster_id so only setting it in pillars
-        # In further steps when we export confstore it would add it to
-        # confstore.
-        PillarSet().run(
-            f'cluster/mgmt_vip',
-            kwargs['mgmt_vip'],
-            local=True)
-
-        node_roles = ['primary'] if kwargs['primary'] else ['secondary']
-        # set node roles
-        PillarSet().run(
-            f"cluster/{node}/roles",
-            node_roles,
-            local=True
-        )
+                    
+        if kwargs['mgmt_vip']:
```

Comment:
can we use kwargs.get('mgmt_vip'). if someone try to call this run function directly without mgmt_vip it will fail

Reply:
Yes. Fixed.